tests/common/cypress/support.d.ts

```diff
@@ -1,10 +1,18 @@
+type Cluster = 'sc' | 'wc'
+type GrafanaRole = 'Admin' | 'Editor' | 'Viewer'
+
+declare const yqArgsToConfigFiles: (cluster: Cluster, expression: string) => string
+declare const userToSession: (user: string) => string
+
 /// <reference types="cypress" />
 
 declare namespace Cypress {
-  type Cluster = 'sc' | 'wc'
-  type GrafanaRole = 'Admin' | 'Editor' | 'Viewer'
-
   interface Chainable<Subject> {
+    /**
+     * Apparently this was removed in Cypress v10...
+     */
+    fail(message: string): void
+
```

Comment:
Feels odd that we would need to manage a type that isn't ours. Do you know why they removed it?

Reply:
That's probably a good idea. That way we don't have to do it on 10x more places when they finally decide to remove the function. :smile: